src/components/InformationPanel/InformationPanel.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { storiesOf } from '@storybook/react'
+import React from 'react'
+import styled from 'styled-components'
+
+import { InformationPanel } from './InformationPanel'
+
+storiesOf('InformationPanel', module).add('all', () => (
+  <Wrapper>
+    <InformationPanel title="パネルタイトル">
+      パネルコンテンツパネルコンテンツパネルコンテンツパネルコンテンツパネルコンテンツパネルコンテンツパネルコンテンツパネルコンテンツパネルコンテンツパネルコンテンツパネルコンテンツパネルコンテンツパネルコンテンツ
+    </InformationPanel>
+  </Wrapper>
+))
```

Comment:
[MUST] Prepare as many stories as possible to understand the difference in behavior depending on props.

Reply:
Should I use english in component description?

Reply:
Japanese is okay 👍

Reply:
I will use something like loremipsum~